src/resources/Account/VerificationSessions.ts

```diff
@@ -0,0 +1,186 @@
+import Resource, { IResourceError } from '../../resource';
+import Configuration from '../../configuration';
+import type {
+  IPlaidBalance,
+  IPlaidTransaction,
+  IMXAccount,
+  IMXTransaction,
+  ITellerBalance,
+  ITellerTransaction,
+} from './externalTypes';
+
+export const AccountVerificationSessionStatuses = {
+  pending: 'pending',
+  in_progress: 'in_progress',
+  verified: 'verified',
+  failed: 'failed',
+};
+
+export type TAccountVerificationSessionStatuses = keyof typeof AccountVerificationSessionStatuses;
+
+export const AccountVerificationSessionTypes = {
+  micro_deposits: 'micro_deposits',
+  plaid: 'plaid',
+  mx: 'mx',
+  teller: 'teller',
+  standard: 'standard',
+  instant: 'instant',
+  pre_auth: 'pre_auth',
+};
+
+export type TAccountVerificaionSessionTypes = keyof typeof AccountVerificationSessionTypes;
+
+export const AccountVerificationCheckStatusTypes = {
+  pass: 'pass',
+  fail: 'fail',
+};
+
+export type TAccountVerificationPassFail = keyof typeof AccountVerificationCheckStatusTypes;
+
+export interface IAccountVerificationSessionMicroDeposits {
+  amounts: number[] | [];
+};
+
+export interface IAccountVerificaitonSessionPlaid {
+  balances: IPlaidBalance | {};
+  transactions: IPlaidTransaction[] | [];
+};
+
+export interface IAccountVerificationSessionMX {
+  account: IMXAccount | {};
+  transactions: IMXTransaction[] | [];
+};
+
+export interface IAccountVerificationSessionTeller {
+  balances: ITellerBalance | {};
+  transactions: ITellerTransaction[] | [];
+};
+
+// TODO: Implement the following interfaces
+export interface IAccountVerificationSessionTrustProvisioner {};
+export interface IAccountVerificationSessionAutoVerify {};
+export interface IAccountVerificationSessionThreeDS {};
+export interface IAccountVerificationSessionIssuer {};
+
+export interface IAccountVerificationSessionStandard {
+  number: string;
+};
+
+export interface IAccountVerificationSessionInstant {
+  exp_year?: string | null;
+  exp_month?: string | null;
+  exp_check?: TAccountVerificationPassFail | null;
+  number?: string | null;
+};
+
+export interface IAccountVerificationSessionPreAuth extends IAccountVerificationSessionInstant {
+  cvv?: string | null;
+  cvv_check?: TAccountVerificationPassFail | null;
+  exp_month?: string | null;
```

Comment:
Nit: This and `exp_year` already exist on the interface that's getting extended `IAccountVerificationSessionInstant`